functional/adoptopenjdk/testng.xml

```diff
@@ -0,0 +1,23 @@
+<!--
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+-->
+
+<!DOCTYPE suite SYSTEM "http://testng.org/testng-1.0.dtd">
+<suite name="AdoptOpenJDK Tests" parallel="none" verbose="2">
+    <test name="AdoptOpenJDKTests">
```

Comment:
As Lan suggested, we can separate the tests into different names:

<test name="FeaturesArePresentTests">
   <classes>
            <class name="net.adoptopenjdk.test.CheckXXOptionAvailableTest"/>
 </classes>
    </test>
<test name="CheckPropertiesTests">
   <classes>
            <class name="net.adoptopenjdk.test.VendorTest"/>
 </classes>
    </test>

In hindsight, it would have made some sense to group the ssl/cert test in with CheckXXOptionAvailableTest as a set of FeaturesArePresentTests or something along those lines.  If you already have some other tests in mind, can we lay out a plan for other checks that should be added, so we think about how those new tests will be organized alongside the existing ones?

Reply:
we still can move them together, by moving the contents of functional/security and the contents of functional/adoptopenjdk into functional/packaging, then add this to the testng.xml file

```
	<test name="SecurityTests">
		<classes>
			<class name="net.adoptopenjdk.test.DefaultTlsFunctionalTest"/>
			<class name="net.adoptopenjdk.test.Tls13FunctionalTest"/>
		</classes>
	</test>
```

and functional/packaging/playlist.xml contains 3 targets, one for SecurityTests, one for FeaturesArePresentTests and one for CheckPropertiesTests.  

Note: often each test target in a playlist contains 100s of test cases.  When we structure targets to be all approximately the 'same size / same execution time', it makes for optimal scheduling and parallelization within a CI system.

Reply:
I'm a bit at a loss here. Is the deciding factor the runtime or the purpose of the tests? The security tests can be run against any OpenJDK and should be successful. The AdoptOpenJDK tests are specific for AdoptOpenJDK. But yeah, it's sometimes hard to decide whether it's a general purpose test or AdoptOpenJDK specific: Shenandoah is not in Oracle builds (it might even be absent from downloads from jdk.java.net), but present in everything built by Red Hat (even 8). In 11.0.9, it requires a compiler flag, but not in 15+. JFR is absent from the upstream binaries we host (at least from some). I don't know whether it requires a flag.

So from my POV it makes sense to have different test groups.